src/AudioAnalysisTools/DSP/FrequencyScale.cs

```diff
@@ -396,9 +396,16 @@ public static void DrawFrequencyLinesOnImage(Bitmap bmp, int[,] gridLineLocation
             for (int b = 0; b < bandCount; b++)
             {
                 int y = height - gridLineLocations[b, 0];
+                int hertzValue = gridLineLocations[b, 1];
+                if (bmp.Width < 120)
+                {
+                    // convert label to kHz so does not hide too much spectrogram
+                    hertzValue = hertzValue / 1000;
+                }
+
```

Comment:
Please revert this change.

We aim for consistency in the images we output. If an image is too short then it is not well suited to a LD spectrogram.

Either all values on all images are in Hz, or in KHz, not both